packages/repo-tools/src/commands/openapi/client/generate.ts

```diff
@@ -62,10 +61,8 @@ async function generate(spec: string, outputDirectory: string) {
     ],
     {
       maxBuffer: Number.MAX_VALUE,
-      cwd: cliPaths.ownDir,
```

Comment:
Testing this locally, I think this is causing the command to create a root `openapitools.json` file instead of using the `repo-tools/openapitools.json`.